lib/absinthe/relay/node/helpers.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Absinthe.Relay.Node.Helpers do
+
+  alias Absinthe.Relay.Node
+
+  @doc """
+  Wrap a resolver to parse node (global) ID arguments before it is executed.
+
+  ## Examples
+
+  Parse a node (global) ID argument `:item_id` (which should be an ID for the
+  `:item` type)
+
+  ```
+  resolve parsing_node_ids(&my_field_resolver/2, TheSchema, item_id: :item)
+  ```
+  """
+  def parsing_node_ids(resolver, schema, id_keys) do
```

Comment:
@benwilson512 Good point, we do! https://github.com/absinthe-graphql/absinthe/blob/master/lib/absinthe/execution/field.ex#L17 -- I'd forgotten.

I'll fix this.